### ISSUE TITLE
Check random bugs reported by discord user

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3077]
getallgroups resulting in wapi error??

```
{
  "success": false,
  "error": {
    "name": "Error",
    "message": "Please set arguments in request json body, not in params. Evaluation failed: TypeError: WAPI[("get" + _0x12b3de(...) + _0x12b3de(...) + "tsW" + _0x12b3de(...) + _0x12b3de(...) + "Msg")](...)[_0x12b3de(...)] is not a function\n    at window.<computed>.<computed> [as getAllChatsWithMessages] (pptr://puppeteer_evaluation_script:1:483356)\n    at pptr://puppeteer_evaluation_script:1:29"
  }
}
```

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.59.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.59.0...4.59.1) (2023-03-23)

- 🐛 fix #3072 [`#3072`](https://github.com/open-wa/wa-automate-nodejs/issues/3072)
- Release 4.59.1 [`1fcf9e6`](https://github.com/open-wa/wa-automate-nodejs/commit/1fcf9e66ba6a38e40ca8b8b450aa3ab157ef3577)
- 🥚 patches update [`548644d`](https://github.com/open-wa/wa-automate-nodejs/commit/548644dfc5c052e3992dccdcf87752390e48e92d)
- updated types-only package [`c122f4c`](https://github.com/open-wa/wa-automate-nodejs/commit/c122f4c80ec00168a174d0c25684ca0d8a7f5199)

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.61.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.59.1...4.61.0) (2023-03-26)

- Release 4.61.0 [`ef166d1`](https://github.com/open-wa/wa-automate-nodejs/commit/ef166d18141481a9f15143a47ad1649c89374d24)
- 🗑️ ALL BUTTONS DEPRECATED TILL FURTHER NOTICE 🤬🤬🤬 [`1d75631`](https://github.com/open-wa/wa-automate-nodejs/commit/1d756310181957f55289f73855ae6b8b062e0e4e)
- ⚡️ optimize `getAllGroups` #3077 [`0347d3c`](https://github.com/open-wa/wa-automate-nodejs/commit/0347d3c7e456c7802388695941af15e1be0552f1)
- updated types-only package [`cdd7a3e`](https://github.com/open-wa/wa-automate-nodejs/commit/cdd7a3ea111c42611f6bbb6ef4db951c592c4ae2)
- 💚 fix build issues [`2c53743`](https://github.com/open-wa/wa-automate-nodejs/commit/2c537439960d7473b717b8bceb74f25dcc15a2ed)
- 🥚 patches update [`3e9a3d7`](https://github.com/open-wa/wa-automate-nodejs/commit/3e9a3d73b819d3a08059a828ae89881cb873e915)
- 🥚 patches update [`5fbac23`](https://github.com/open-wa/wa-automate-nodejs/commit/5fbac232a140884a73770ccf0a63c908573ee48d)
